kafka/record/default_records.py

```diff
@@ -185,6 +188,8 @@ def _maybe_uncompress(self):
                     uncompressed = snappy_decode(data.tobytes())
                 if compression_type == self.CODEC_LZ4:
                     uncompressed = lz4_decode(data.tobytes())
+                if compression_type == self.CODEC_ZSTD:
+                    uncompressed = zstd_decode(data)
```

Comment:
Should this be `data.tobytes()`? Just looking at the other codecs above.

Reply:
> Should this be `data.tobytes()`? Just looking at the other codecs above.

it works with both, since it's passed directly to the c library, which is compatible with the `memoryview` type, but consistency wise it should be as the calls above